src/test/java/com/katelocate/menugenerator/RecipeRepositoryTests.java

```diff
@@ -24,7 +24,7 @@
 
 import java.sql.ResultSet;
 import java.sql.SQLException;
-import java.util.List;
+import java.util.*;
```

Comment:
Why is this change needed?

Reply:
I relied on the IntelliJ suggestion. In this file I use List, Set and HashSet classes that are all in the java.util lib, so it decided to switch to this form of import.